css/style.css

```diff
@@ -62,12 +60,20 @@ body {
   padding: 5px;
 }
 
+.view-btn li {
+  margin: 0 8px;
+  background-color: #36b37e;
+  color: #fff;
+  padding: 8px;
+  border-radius: 4px;
+}
+
 .nav-menu {
-  display: none;
-  width: 100%;
-  align-items: center;
+  display: flex !important;
+  width: 80%;
   text-align: center;
-  justify-content: center;
+  justify-content: space-between !important;
+  margin: 24px auto;
 }
 
```

Comment:
- [ ] For the styling Please add the correct margin-top value to the nave links on the mobile menu to match the positioning in the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/168853247-4a0159ea-136f-4ff2-8519-360878a37663.png)  | ![image](https://user-images.githubusercontent.com/68796232/168853194-06d9ef7c-b4d7-4155-87a9-6af733ac2fa2.png)  |